lms/templates/instructor/instructor_dashboard_2/cohort-discussions.underscore

```diff
@@ -0,0 +1,52 @@
+<div class="cohort-discussions-nav">
+    <h3 class="subsection-title"><%- gettext('Specify whether discussion topics are divided by cohort') %></h3>
+    <form action="" method="post" id="cohort-course-wide-discussions-form" class="cohort-course-wide-discussions-form">
+        <div class="wrapper cohort-management-supplemental">
+            <div class="form-fields">
+                <div class="form-field">
+                    <div class="course-wide-discussion-topics">
+                        <h3 class="subsection-title"><%- gettext('Course-wide Discussion Topics') %></h3>
```

Comment:
@muzaffaryousaf My apologies for introducing this inconsistency!
"Course-wide" should have uppercase "W" as well: "Course-Wide Discussion Topics". Thanks!